### PR TITLE
added mobile nav bar

### PR DESCRIPTION
**Pull request by bvillaroman**

### CODE REVIEW

File: src/components/Navbar/NavBar.tsx [hunk 1]
```diff
@@ -1,25 +1,82 @@
...
+  };
+
+  return (
+    <div className={styles.navBarContainer}>
+      <img className={styles.image} src={sclLogo} alt="Logo" />
+      {hamburgerOpen ? (
+        showCompactMenu()
+      ) : (
+        <div className={styles.navLinksContainer}>
+          <Link to="/" className={styles.navLink}>
```
Comment (jtmckibb) [inline src/components/Navbar/NavBar.tsx:69]: This should link to https://sendchinatownlove.com/

File: src/components/Navbar/NavBar.tsx [hunk 2]
```diff
@@ -1,25 +1,82 @@
...
+            HOME
+          </Link>
+          <Link to="/sellers" className={styles.navLink}>
```
Comment (jtmckibb) [inline src/components/Navbar/NavBar.tsx:72]: We should update this route to be merchants/

File: src/components/Navbar/NavBar.tsx [hunk 3]
```diff
@@ -1,25 +1,82 @@
...
+            MERCHANTS
+          </Link>
+          <Link to="/about" className={styles.navLink}>
```
Comment (jtmckibb) [inline src/components/Navbar/NavBar.tsx:75]: I don't think we have this page implemented in Weebly so you can remove it here.

**Comment by jtmckibb** [comment]
Looks good with some nits

**Comment by jtmckibb** [comment]
I also realized that our 404 page has the wrong links in it XD

File: src/components/Navbar/NavBar.tsx (hunk 1 above)
Comment (bvillaroman) [reply src/components/Navbar/NavBar.tsx:69]: the / is wherever the index lives in (when deployed should be sendchinatownlove.github.io)

File: src/components/Navbar/NavBar.tsx (hunk 3 above)
Comment (bvillaroman) [reply src/components/Navbar/NavBar.tsx:75]: the path for all of our merchants is currently /sellers right now. I could change the path, but i would rather do that in a new branch

File: src/components/Navbar/NavBar.tsx (hunk 1 above)
Comment (jtmckibb) [reply src/components/Navbar/NavBar.tsx:69]: yeah but our home page is hosted by Squarespace which lives under the main domain, while this lives under the merchants subdomain (which is hosted by GitHub). We plan to delete the existing home page under merchants. It shouldn't be accessible and should be replaced with a 404

Comment (bvillaroman) [reply src/components/Navbar/NavBar.tsx:69]: ah ok, will change here then

**Pull request merged**